includes/class-sensei-wc.php

```diff
@@ -1490,7 +1490,7 @@ public static function course_update( $course_id = 0, $order_user = array(), $or
 		 * for WOOCOMMERCE_CHECKOUT. If we're in the WooCommerce checkout
 		 * process, we can trust that nonce verification has already been done.
 		 */
-		if ( WOOCOMMERCE_CHECKOUT ) {
+		if ( defined( WOOCOMMERCE_CHECKOUT ) && WOOCOMMERCE_CHECKOUT ) {
```

Comment:
`defined()` argument doesn't have to be in quotes?

Reply:
Whoops! Should be fixed in https://github.com/Automattic/sensei/pull/2354